docs/developers/Centaur.md

```diff
@@ -74,5 +90,15 @@ The `testFormat` field can be one of the following, case insensitive:
 * `workflowfailure`: The workflow being supplied is expected to fail
 
 The `metadata` is optional. If supplied, Centaur will retrieve the metadata from the successfully completed workflow and compare the values retrieved to those supplied. At the moment the only fields supported are strings, numbers and booleans.
-For any metadata values which require workflow ID (i.e, file paths), use <\<UUID>> as a placeholder instead. For example:
-* "calls.hello.hello.stdout": "gs://google-project/jes/root/wdl/<\<UUID>>/call-task/task-stdout.log"
\ No newline at end of file
+
+You can find which metadata is recorded by running a workflow ```java -jar <Cromwell JAR> run -m metadata.json my_workflow.wdl```.
+This will save the metadata in `metadata.json`.
+
+For any metadata values or outputExpectations which require workflow ID (i.e, file paths), use `<<UUID>>` as a placeholder instead. For example:
+* `"calls.hello.hello.stdout": "gs://google-project/jes/root/wdl/<<UUID>>/call-task/task-stdout.log"`
+
+In case the absolute path the cromwell root is used (for example: `/home/my_user/projects/cromwell/cromwell-executions`)
+ you can use `<<WORKFLOW_ROOT>>` as a replacement. 
+* `"calls.hello.hello.exit_code": "<<WORKFLOW_ROOT>>/call-hello/execution/exit_code"`
+
+TODO: `<<CACHE_HIT_UUID>>` In what sort of tests can you use this?
```

Comment:
Call caching tests! 😄 These magic values should expand to the cache hit workflow's ID.